ansible/roles/zookeeper/templates/zoo.cfg

```diff
@@ -16,7 +16,9 @@ clientPort={{ zookeeper_client_port }}
 # increase this if you need to handle more clients
 maxClientCnxns=100
 # support zk monitoring (for zk >= 3.5)
-4lw.commands.whitelist=mntr
+{% if zookeeper_version.startswith('3.5') %}
```

Comment:
There seems to be a better way in Ansible to compare versions. Specifically we can use the below instead of the string approach:

```zookeeper_version is version('3.5', '>=')```

I did a quick unit test with the below playbook and it works as intended, including minor version handling.
```
- hosts: localhost
  gather_facts: no
  tasks:
          - set_fact:
                  zookeeper_version: "3.5.6"
          - debug:
                  msg: "needs whitelist"
            when: zookeeper_version is version('3.5', '>=')
```